source/hic_hal/nxp/lpc11u35/gpio.c

```diff
@@ -94,11 +94,13 @@ void gpio_init(void)
     PIN_CDC_LED_IOCON = PIN_CDC_LED_IOCON_INIT;
     LPC_GPIO->SET[PIN_CDC_LED_PORT] = PIN_CDC_LED;
     LPC_GPIO->DIR[PIN_CDC_LED_PORT] |= PIN_CDC_LED;
+#ifndef GPIO_RESET_IN_NOT_SUPPORTED
     // configure Button(s) as input
     PIN_RESET_IN_IOCON = PIN_RESET_IN_IOCON_INIT;
```

Comment:
Is GPIO_RESET_IN_NOT_SUPPORTED needed?  This is on pin PIO0_1 which is a no-connect for the LPC4088DM.  This pin will have its pullup enabled, so it should be fine to leave it as a no connect.